autolab/Autolab
# Submission Version Switching

## Description
- Add buttons to switch to prev / next version of a student's submission that contains a specific file
- Make buttons update when switching between files

## Motivation and Context
When investigating AIVs, instructors might want to track changes to a specific file across the different versions of a student's submission. This PR adds buttons to enable instructors to easily switch between the different versions.

**Note**: For single file handin formats, the arrows will effectively work to cycle through the versions.

## How Has This Been Tested?
- Create a new assessment with handin format `handin.tar`
- Submit `hello.tar`, as well as modified versions (e.g. remove files, add files)
- Ensure that prev / next buttons work as expected (i.e. that they go to the same file, skip versions where the file doesn't exist)
- Ensure that when we change the file selected, the buttons update as expected
- Also check that for single file handin formats (e.g. `.c`, `.pdf`) that the arrow buttons serve to enumerate through the versions

<img width="1440" alt="Screen Shot 2022-07-17 at 20 46 57" src="https://user-images.githubusercontent.com/9074856/179399001-eedaa8c6-f78b-4ef2-a597-5cdf92459e84.png">

## Types of changes

## Checklist:
